src/main/java/Config/Config.java

```diff
@@ -23,6 +46,7 @@ public static Routes setRoutes() {
     routes.add(new Route(HEAD, "/", new HEAD200Handler()));
     routes.add(new Route(HEAD, "/foobar", new HEAD404Handler()));
     routes.add(new Route(GET, "/redirect", new RedirectHandler()));
+    routes.add(new Route(GET, "/logs", new AUTHHandler()));
```

Comment:
Just a thought, but I see you're using `Config.credential` in `AUTHHandler`'s `handle` method, but here you initialize it.  It might decouple things a bit if you were _passing_ `Config.credential` in as an initialization argument to the handler here, and it might make it easier to know that the dependency is there.